firmata/readme.md

```diff
@@ -0,0 +1,46 @@
+# Usage
+
+For typescript projects, instead of
+
+```js
+var Board = require('firmata');
+```
+
+you should use
+
+```typescript
+import * as Board from 'firmata'
+```
+
+You will find that `const enums` are exported under the namespace `Board` for ease of comparison or
+use with other libraries before any boards are actually instantiated.
+
+It's also important to note that if you want to extend the `Board` class in typescript, the
+following will work (as long as you turn on `--noImplicitUseStrict` when compiling); here's
```

Comment:
@andy-ms Any ideas on why `"use strict"` in js-code generated from typescript might cause run-time class-extensions issues? Any recommendations on how to test before it gets published and I can do `npm i -D @types/firmata`?